pandas/tests/plotting/test_datetimelike.py

```diff
@@ -519,14 +519,14 @@ def test_finder_hourly(self):
             xp = Period('1/1/1999', freq='H').ordinal
         assert rs == xp
 
+    @td.skip_if_mpl_1_5
     @pytest.mark.slow
     def test_gaps(self):
         ts = tm.makeTimeSeries()
         ts[5:25] = np.nan
         _, ax = self.plt.subplots()
         ts.plot(ax=ax)
         lines = ax.get_lines()
```

Comment:
yeah, we occasionally do this. if its actually important then could split into 2 tests. not a big deal here.